SECURITY.md

```diff
@@ -131,7 +131,7 @@ or mitigation so that a realistic timeline can be communicated to users.
 
 **Disclosure of Forthcoming Fix to Users** (Completed within 1-7 days of Disclosure)
 
-- The Fix Lead will email [envoy-announce@googlegroups.com](https://groups.google.com/forum/#!forum/envoy-announce)
+- The Fix Lead will email [envoy-security-announce@googlegroups.com](https://groups.google.com/forum/#!forum/envoy-security-announce)
```

Comment:
Does envoy-security-announce include envoy-announce? (Will people miss the security disclosure if they previously relied on envoy-announce emails)

Reply:
No, googlegroups does not allow me to add group to another group. I get
"Some addresses are groups, but groups may not be added to other groups."

Reply:
Maybe both then?

Reply:
Big +1 for both. I think we should always at least cc envoy-announce@ even if the primary target is envoy-security-announce@

Reply:
(Oops looks like you are already CCing envoy-announce@ in this PR. I'm fine with that or on TO. Up to you and @asraa)

Reply:
done